### DISCUSSION TITLE
Welcome, DevElkami!

### DISCUSSION THREAD

**Discussion by DevElkami** [ParanoidUser/yolo#636] [Subscribe]
Can I with Contributor permissions to make changes to the repository ?

**Comment by ParanoidUser** [comment]
Ahoy @DevElkami, welcome aboard!

Thank you for your interest in this project! Sure, you can! See more details about Collaborator actions [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/permission-levels-for-a-personal-account-repository#collaborator-access-for-a-repository-owned-by-a-personal-account). The invitation to join the collaborator group has been sent to you. Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Cheers
